wiki/Beatmap_Spotlights/Seasons/2021_Spring/en.md

```diff
@@ -80,3 +152,27 @@ As of now, joining the weekly multiplayer lobbies requires participants to downl
 - [Tigran Hamasyan - What the Waves Brought (Cut Ver.) (Parachor) \[Rising Tide\]](https://osu.ppy.sh/beatmapsets/1081010#mania/2261508)
 - [The Black Dahlia Murder - Deathmask Divine (Davvy) \[April & Tim's Necromance\]](https://osu.ppy.sh/beatmapsets/1363029#mania/2819895)
 - [John Wasson - Caravan (Evening) \[drown\]](https://osu.ppy.sh/beatmapsets/1086739#mania/2272532)
+
+#### Playlist B
+
+- [DIVERSA - i use raw expectation to unlock placebo. the placebo is the next thing i do. (Parachor) \[.\_//- [Advanced]\]](https://osu.ppy.sh/beatmapsets/610265#mania/1288556)
```

Comment:
This diff name is funky, need someone who is more versed in escape characters to see how `._//- [Advanced]` should actually be.

Reply:
`.\_//\- \[Advanced\]` should do the trick.

Reply:
you don't escape the hyphen do you?  `.\_//- \[Advanced\]`
```suggestion
- [DIVERSA - i use raw expectation to unlock placebo. the placebo is the next thing i do. (Parachor) \[.\_//- \[Advanced\]\]](https://osu.ppy.sh/beatmapsets/610265#mania/1288556)
```

Reply:
It's optional, in a way. You are allowed to escape it in most Markdown specs, however in this case it will produce the same output since the hyphen is not at the start of the line and therefore doesn't create a bulleted list item.

Reply:
it's also fine as-is.